pkgs/servers/misc/navidrome/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl }:
```

Comment:
```suggestion
{ stdenv, fetchzip }:
```

Reply:
I can't do this since I can't use `fetchzip`